packages/sdks/nextjs-sdk/test/server/authMiddleware.test.ts

```diff
@@ -79,6 +79,72 @@ describe('authMiddleware', () => {
 		expect(NextResponse.redirect).not.toHaveBeenCalled();
 	});
 
+	it('redirects unauthenticated users for private routes', async () => {
+		mockValidateJwt.mockRejectedValue(new Error('Invalid JWT'));
+		const middleware = authMiddleware({
+			privateRoutes: ['/private']
+		});
+		const mockReq = createMockNextRequest({ pathname: '/private' });
```

Comment:
I think that it is important to have negative test - if user hit private route it is get redirected